app/services/game_wizard_service.rb

```diff
@@ -84,21 +84,11 @@ def persist_step
       end
       cache_key
     when 4
-      #If user uploaded new poster
       if poster.present?
-        raw_ext = extract_poster_ext(poster)
-        if raw_ext
-          poster_ext = raw_ext.split('/').last
-        else
-          #Specially to cause error on poster validation (wrong format)
-          poster_ext = 'txt'
-        end
-
-        poster_tmp_url = tmpfilename = Rails.root.join('tmp', 'cache', "#{cache_key}_poster.#{poster_ext}")
-        File.open(tmpfilename,'w') do |file|
-          file.write Base64.decode64(poster.sub("data:#{raw_ext};base64,",'')).force_encoding('UTF-8')
+        poster_tmp_url = Rails.root.join('tmp', 'cache', poster.original_filename)
+        File.open(poster_tmp_url ,'w') do |file|
```

Comment:
Space missing after comma.<br>Space found before comma.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.